src/store/models/network.ts

```diff
@@ -178,6 +179,14 @@ export interface NetworkModel {
   setAutoMineMode: Action<NetworkModel, { id: number; mode: AutoMineMode }>;
   setMiningState: Action<NetworkModel, { id: number; mining: boolean }>;
   mineBlock: Thunk<NetworkModel, { id: number }, StoreInjections, RootModel>;
+  setDockerExternalNetworkName: Thunk<
+    NetworkModel,
+    { id: number; externalNetworkName: string | undefined },
+    StoreInjections,
+    RootModel
+  >;
+  getExternalDockerNetworks: Thunk<NetworkModel, void, StoreInjections, RootModel>;
```

Comment:
Use `Thunk<NetworkModel, void, StoreInjections, RootModel, Promise<string[]>>` so that the return value is typed to `string[]`.